tests/headers/structs.h

```diff
@@ -32,9 +32,28 @@ typedef struct tagWNDCLASSEXA {
 } WNDCLASSEXA, *PWNDCLASSEXA, near *NPWNDCLASSEXA, far *LPWNDCLASSEXA;
 
 // Test altering the packing of a structure.
-#pragma pack(16)
+#pragma pack(push, 16)
```

Comment:
push is optional (at least under visualc) so we must be sure that the case is covered somewhere else.